### DISCUSSION TITLE
Any way to isolate backing/multiple/choir vocals?

### DISCUSSION THREAD

**Discussion by Eddy119** [Anjok07/ultimatevocalremovergui#586] [Q&A]
What the title says...

**Comment by ajaxxxxxxx** [comment]
I too would love an answer to this

**Reply by Eddy119** [reply]
I found out using Kim Vocal isolates the backing vocals too while using Karaoke only isolates the lead vocals. So using one after the other seems to work sometimes (not all the time). If anyone has better methods, please share.

**Reply by datsenprinting** [reply]
kim 1 or 2?

**Reply by deadniell** [reply]
6_HP_Karaoke_UVR... or... MDX NET Karaoke?

**Reply by datsenprinting** [reply]
After some testing what works best for me is Kim 2, then 5_HP_ Karaoke or if that doesn't work 6_HP_Karaoke.

**Reply by deadniell** [reply]
indeed Kim 2 is surprisingly great, gonna try both those 5 and 6 HP, thankss

**Reply by datsenprinting** [reply]
Of course! Check out my ai cover page if you get a chance https://www.youtube.com/@aiarticons/videos
